### PR TITLE
[APPWIZ] Improve Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

**Review by JoachimHenze — APPROVED** [review]

File: dll/cpl/appwiz/lang/ro-RO.rc [hunk 1]
```diff
@@ -29,32 +29,32 @@ END
 
 IDD_SHORTCUT_LOCATION DIALOGEX 0, 0, 340, 200
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Expertul pentru creare de scurtături"
+CAPTION "Creare comandă rapidă"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    LTEXT "Acest expert vă va ajuta în crearea unei scurtături\ncătre adrese din Internet, calculatoare, dosare, fișiere\nlocale sau din rețea.", -1, 120, 15, 200, 30
-    LTEXT "Scrieți adresa elementului:", -1, 120, 50, 120, 10
+    LTEXT "Acest asistent vă va ajuta \în crearea unei scurtături\ncătre adrese din Internet, calculatoare, dosare, fișiere\nlocale sau din rețea.", -1, 120, 15, 200, 30
```
Comment (JoachimHenze) [inline dll/cpl/appwiz/lang/ro-RO.rc]: what is the purpose of that added backslash? Why wasn't it necessary before?

Comment (JoachimHenze) [reply dll/cpl/appwiz/lang/ro-RO.rc]: resolved.

Comment (AndreiMiloiu) [reply dll/cpl/appwiz/lang/ro-RO.rc]: I added it by mistake.

**Pull request merged**